src/blocks/Info/README.md

```diff
@@ -2,14 +2,18 @@ Info block
 
 `type: 'info-block'`
 
-`title: string` — Title on the right
+`title: string` — Title on the right. DEPRECATED
 
-`sectionsTitle: string` — Title on the left
+`sectionsTitle: string` — Title on the left. DEPRECATED
 
 `backgroundColor?: string` — Fill color
 
-`buttons?: Button[]` — Buttons on the left
+`buttons?: Button[]` — Buttons on the left. DEPRECATED
```

Comment:
please mark this props with `/** @deprecated **/` in code

Reply:
add